src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -478,7 +470,13 @@ public async Task<List<ClusterTransactionCommand.SingleClusterDatabaseCommand>>
                 ClusterTransactionCommand.ReadCommandsBatch(context, Name, fromCount: _nextClusterCommand, take: batchSize));
 
             if (batch.Count == 0)
+            {
+                var record = _serverStore.Cluster.ReadDatabase(context, Name, out _);
+                var index = _serverStore.Cluster.GetLastCompareExchangeIndexForDatabase(context, Name);
+                
+                NotifyFeaturesAndListenersAbout(record, index);
```

Comment:
In many cases, this isn't changed. But we'll still invoke:

```
ThreadingHelper.InterlockedExchangeMax(ref LastModifiedIndex, index);
_notifiedListeners.SetAndResetAtomically();
```

Which can cause threads to wake up. Let's add a check that would skip if if this is the same thing.

Reply:
Done.